packages/wrangler/package.json

```diff
@@ -88,6 +88,7 @@
 		"@esbuild-plugins/node-globals-polyfill": "^0.1.1",
 		"@esbuild-plugins/node-modules-polyfill": "^0.1.4",
 		"blake3-wasm": "^2.1.5",
+		"create-cloudflare": "^1.0.0",
```

Comment:
let's not have a dependency on this, and use npx instead here? if not, let's move this to devDependencies.